.github/workflows/build.yml

```diff
@@ -20,72 +20,72 @@ jobs:
     steps:
       - name: Checkout repository and submodules
         # See https://github.com/actions/checkout/commits
-        uses: actions/checkout@72f2cec99f417b1a1c5e2e88945068983b7965f9
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
         with:
           submodules: recursive
 
       - name: Validate Gradle Wrapper
         # See https://github.com/gradle/wrapper-validation-action/commits
-        uses: gradle/wrapper-validation-action@56b90f209b02bf6d1deae490e9ef18b21a389cd4
+        uses: gradle/wrapper-validation-action@699bb18358f12c5b78b37bb0111d3a0e2276e0e2 # v2.1.1
 
         # See https://github.com/actions/setup-java/commits
-      - uses: actions/setup-java@4075bfc1b51bf22876335ae1cd589602d60d8758
+      - uses: actions/setup-java@387ac29b308b003ca37ba93a6cab5eb57c8f5f93 # v4.0.0
         with:
           java-version: 17
           distribution: temurin
           
       - name: Build
         # See https://github.com/gradle/gradle-build-action/commits
-        uses: gradle/gradle-build-action@3bfe3a46584a206fb8361cdedd0647b0c4204232
+        uses: gradle/actions/setup-gradle@417ae3ccd767c252f5661f1ace9f835f9654f2b5 # v3.1.0 from https://github.com/gradle/actions/commits
```

Comment:
Commit from https://github.com/gradle/gradle-build-action/commits is unable to be found, use https://github.com/gradle/actions/commits instead.

Reply:
What do you mean with "cannot be found"?

I can see the commits (and it's hash) just fine:
![Screenshot_2024-02-24-12-09-53-64_3aea4af51f236e4932235fdada7d1643](https://github.com/GeyserMC/Geyser/assets/105284508/aeaceef8-7d97-46be-9733-e81e8ae0c824)

Reply:
![image](https://github.com/GeyserMC/Geyser/assets/16631550/64969b5c-cff6-4d65-a331-480a71462bdc)
![image](https://github.com/GeyserMC/Geyser/assets/16631550/0245121f-3a30-4cfa-a93a-c9065321ad1e)  

When setting to [gradle-build-action's](https://github.com/gradle/gradle-build-action/commits/) commit encountered error that action cannot find the commit